languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/hunspell/spelling.txt

```diff
@@ -60969,3 +60969,5 @@ Altersgebrechlichkeit
 Arabischkenntnisse/N
 Alhamdulillah
 Ben-Gurion/S #name
+österr. #abk
+gemeindt. #abk
```

Comment:
nicht sicher, ob die anderen ABKs auch für DE sind?

Reply:
Muss man sie doppelt hinzufügen? Werden sie in de-CH nicht automatisch auch akzeptiert, wenn sie in de-DE spelling hinzugefügt werden? Falls wir da nicht sicher sind, können wir die beiden in spelling.txt erstmal in spelling-de-CH.txt weglassen und es morgen testen, ob sie schon akzeptiert werden wenn im Editor de-CH eingestellt ist...